src/page-multi-facility/HistoricalCasesChart/index.tsx

```diff
@@ -23,14 +26,20 @@ function generateBarChartData(
     });
 
     if (existingVersion) {
+      const cases = totalConfirmedCases(existingVersion);
+      // This is necessary since semiotic doesn't have an option to display a stacked bar chart overlay.
```

Comment:
sorry to be pedantic but FWIW Semiotic's api for stacking bars is correct, it is actually we who are abusing the concept slightly by identifying the second bar as "total population" rather than "uninfected population) ... the height of all stacked bars should add up to 100% of whatever the underlying total is, their contents should never overlap. So the calculations you did here are totally sensible and consistent with the chart form we are using (excepting the issue Josh mentions below)

Reply:
True, a more appropriate way of stating this is that semiotic doesn't support overlay bars, which is what we want. I'll update the comment.